conformance/utils/roundtripper/roundtripper.go

```diff
@@ -116,7 +116,7 @@ func (d *DefaultRoundTripper) CaptureRoundTrip(request Request) (*CapturedReques
 
 	if d.Debug {
 		dump, dumpErr := httputil.DumpResponse(resp, true)
```

Comment:
Why not reuse err so we don't have to keep track of stuff? Does the value in err live past this block? (would be a maintenance issue)

Reply:
It takes a bit more code to avoid error shadowing (have to declare dump separately). I don't feel strongly either way though, so adjusted the code to use separate declarations + reuse err var.